src/config.h.in

```diff
@@ -1 +1,4 @@
+#define SETTINGS_FILE "mixxx.cfg"
```

Comment:
Why not keep the `MIXXX_` prefix for both defines? They could otherwise get into conflict with imported defines or symbols from external libaries.

Reply:
Done.